app/models/restaurant.rb

```diff
@@ -1,4 +1,11 @@
 class Restaurant < ApplicationRecord
+
+
+  def self.search(search)
+    where("name ilike ?", "%#{search}%")
+    # where(dollar_value: search)
```

Comment:
I am not wholly sold on this as a pattern (esp. with the commented-out code just beneath it, which as a best practice should not be merged into `master`.)